### DISCUSSION TITLE
TSC Meeting May 2022

### DISCUSSION THREAD

**Discussion by lgirdwood** [thesofproject/sof#7400] [Team Posts]
@thesofproject/steering-committee  Now that v2.1 is out let discuss the plans for v2.2 and discuss opens and agenda items. 

Date: Wednesday 11th May 2022 at 17.00 UK Time (BST)
Doodle:https://doodle.com/meeting/participate/id/bmZV6JGd
Meeting will be 1 hour, but may take longer if topics require more discussion and alignment.

**Agenda** (pls add topics)
1) Zephyr status and proposed next steps.
2) Module API updates and proposed next steps
3) Kernel status and proposed next steps.
4) Kconfig build All option
5) Build-able 3rd party module wrappers using stubs
6) CODEOWNERs updates for better reviews. e.g. need some aliases for teams like team-intel, team-amd, team-mtk, team-nxp and so on.
7) SPARSE support - lots of warning being reported now for us to fix.
8) DMA based scheduling - needs better Zephyr integration with Zephyr premption task levels.

**Comment by cujomalainey** [comment]
If we do it the week of May 15th we don't have to worry about @bzhg and I calling in from NA.

**Comment by dbaluta** [comment]
@lgirdwood @cujomalainey So are we good with trying to schedule the meeting for the week of May 15th?
 Not sure what NA means :)

**Comment by cujomalainey** [comment]
NA-> North America

**Comment by dbaluta** [comment]
@lgirdwood @cujomalainey all right. Doodle created (https://doodle.com/meeting/participate/id/bmZV6JGd).

**Comment by lgirdwood** [comment]
Looks like Wednesday 10th at 5 - 6pm UK time has most votes (at 8). Will send invite.
